app/views/users/edit.html.haml

```diff
@@ -0,0 +1,8 @@
+- content_for(:page_title) do
+  %h1 Edit Profile
+
+= simple_form_for @user, url: '/user', method: :put  do |f|
```

Comment:
The URL shouldn't be hardcoded here.  You should use the URL helpers if you need to generate a url.